index.html

```diff
@@ -0,0 +1,86 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css" />
+  </head>
+
+  <body>
+  
+    <div class="sectionWrapper">
+      <header id="header">
+        <nav id="nav-bar">
+          <div class="logo">
+            <img src="./assets/product-landing-page-logo.png" alt="" id="header-img">
+          </div>
+          <div class="nav-list">
+            <ul>
+                <li class="nav-link">
+                  <a href="#features">Features</a>
+                </li>
+                <li class="nav-link">
+                  <a href="#how-it-works">How it Works</a>
+                </li>
+                <li class="nav-link">
+                  <a href="#pricing">Pricing</a>
+                </li>
+            </ul>
+          </div>
+        </nav>
+      </header>
+    </div>
+
+    <div class="infoHeader">
+      <div class="infoHeaderWrapper">
+        <h1>Handcrafted, home-made masterpieces</h1>
+
+        <form action="https://www.freecodecamp.com/email-submit" id="form">
+          <input type="email" name="" id="email" placeholder="Enter your email address">
+          <input type="submit" id="submit">Get Started</input>
+        </form>
+        
+      </div>
+    </div>
+
+    <div id="features">
+      <div class="featureContainer">
+        <div class="feature">
+          <div class="featureText">
+            <h3>Premium Materials</h3>
+            <p>Our trombones use the shinest brass which is sourced locally. This will increase the longevity of your purchase</p>
+          </div>
+        </div>
+        <div class="feature">
+          <div class="featureText">
+            <h3>Fast Shipping</h3>
+            <p>We make sure you receive your trombones as soon as we have finished making it. We also provide free returns if you are not satisfied.</p>
+          </div>
+        </div>
+        <div class="feature">
+          <div class="featureText">
+            <h3>Quality Assurance</h3>
+            <p>For every purchase you make, we will ensure there are no damages or faults and we will check and test the pitch of your instrument.</p>
+          </div>
+        </div>
+      </div>
+    </div>
+
+    <div id="how-it-works">
+      <div class="how-it-works-video">
+        <iframe src="https://www.youtube.com/watch?v=y8Yv4pnO7qc&feature=youtu.be" frameborder="0" id="video">
+
+        </iframe>
+      </div>
+    </div>  
+
+    <div id="pricing">
+      
+    </div>
```

Comment:
It seems that you left unused code in your project. Please, consider that leaving unused code in your files may make it difficult to understand and evaluate.